### PR TITLE
Fix jpeg capture and microphone on ML

### PR DESCRIPTION
**Pull request by machenmusik**
Incorporates the following patch commits:
- fix jpeg capture (918e60f and b15c824) - already in master
- update debug-ml.sh entrypoint (f19546d) - already in master
- there is no AudioContext reset, so don't call it (2af5ae2)

### CODE REVIEW

**Comment by machenmusik** [comment]
(Like the suspend-resume branch itself, this also requires the modulesio/LabSound#suspend-resume to be copied into native-audio-deps to compile.)

File: deps/exokit-bindings/webaudiocontext/src/AudioContext.cpp [hunk 1]
```diff
@@ -9,14 +9,16 @@ lab::AudioContext *getDefaultAudioContext(float sampleRate) {
     _defaultAudioContext = lab::MakeRealtimeAudioContext(sampleRate);
 
     atexit([]() {
-      _defaultAudioContext.reset();
+      // NOTE: There was no reset method for AudioContext!
+      //_defaultAudioContext.reset();
```
Comment (modulesio) [inline deps/exokit-bindings/webaudiocontext/src/AudioContext.cpp]: My thinking is we should reset the pointer to call the `AudioContext` destructor as intended. I think the issue is simply using the wrong signature, or wrong C++ version in compiler.

**Comment by machenmusik** [comment]
So if I understand correctly, the intent is to delete and null the pointer, so reset needs to be called with nullptr as an argument, and that should work with whatever the current C++ version in use is?

**Comment by modulesio** [comment]
Correct, we should leave it in and make sure it compiles. Was the problem that it wasn’t compiling?

**Comment by machenmusik** [comment]
It did compile, but seemed to hang the thread that called it.

**Comment by machenmusik** [comment]
The suggested change does not work; the original change (to not call reset) results in functional microphone behavior; the suggested change (to call reset with nullptr) still appears to hang the thread.

**Comment by machenmusik** [comment]
I don't disagree, but that is outside the portions of the code with which I have sufficient familiarity at present.  Perhaps others can have a quick look at some point.

**Comment by modulesio** [comment]
Yup, two other issues before I can loop back to this.

Maybe it makes the most sense to just continue the work from the `suspend-resume` branch?

**Comment by machenmusik** [comment]
Yeah, this is a merge into `suspend-resume`, maybe you want to cherry-pick the three commits from master and not the commented-out reset -- again, sorry, I need to keep my branch intact, as it is functional enough for me at the moment

**Comment by modulesio** [comment]
I rebased so `suspend-resume` contains the first three commits now.

I guess that means this PR can be closed?

**Comment by machenmusik** [comment]
right, and you are planning to pursue a separate, more thorough fix for #477 and #537.

**Comment by modulesio** [comment]
That wass the plan for the suspend-resume branch.

**Pull request closed without merging**